audformat/core/utils.py

```diff
@@ -439,3 +441,87 @@ def to_segmented_index(
     obj = obj.reset_index(drop=True)
     obj.index = index
     return obj
+
+
+def union(
+    objs: typing.Sequence[typing.Union[pd.Index]],
```

Comment:
Why Union?

Maybe:
```suggestion
    objs: typing.Sequence[pd.Index],
```